pytket/extensions/aqt/multi_zone_architecture/architecture.py

```diff
@@ -141,3 +139,28 @@ def get_zone_max_ions(self, zone_index: int) -> int:
         zone = self.zones[zone_index]
         zone_type = self.zone_types[zone.zone_type_id]
         return zone_type.max_ions
+
+    def __str__(self) -> str:
```

Comment:
Is this tested somewhere?

Reply:
I'm using it in a jupyter notebook. Added test that it, in fact returns a string. But not sure if it makes sense to test the exact content of the string at the moment.